frontend/src/components/SamplePage/SamplePage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function SamplePage() {
+  return (
+    <p>Placeholder for routing</p>
```

Comment:
Just wanted to add a note here that we should remove the sample page before we merge this.